### PR TITLE
completed javascript-modal exercise

### PR DESCRIPTION
**Pull request by amyaitran**
js-modal

### CODE REVIEW

File: javascript-modal/main.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+$modalButton.addEventListener('click', handleClickModal);
+$surveyButton.addEventListener('click', handleClickSurvey);
+
+var modalOn = true;
+var surveyOn = false;
+
+function handleClickModal(event) {
+  if (modalOn) {
+    modalOn = false;
+    surveyOn = true;
```
Comment (codingcodymiller) [inline javascript-modal/main.js:40]: These booleans don't seem to be necessary for anything in particular. Both of these event handler functions always do one thing, so you can likely just leave the behavior as the work for the function, and remove the conditionals surrounding them, along with these boolean flags.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work!! One note for you:

File: javascript-modal/main.js (hunk 1 above)
Comment (amyaitran) [reply javascript-modal/main.js]: that's a lot simpler!

**Review by codingcodymiller — APPROVED** [review]
Excellent work!!

**Pull request merged**